apps/mark-scan/accessible-controller/src/controllerd.rs

```diff
@@ -317,9 +328,6 @@ fn main() {
                 match port.read(serial_buf.as_mut_slice()) {
                     Ok(size) => match handle_command(&mut device, &serial_buf[..size]) {
                         Ok(_) => (),
-                        Err(e) if matches!(e, CommandError::UnsupportedButton(_)) => {
-                            println!("[warn] {e}")
-                        }
                         Err(e) => {
                             eprintln!("Unexpected error handling command: {e}")
                         }
```

Comment:
Since there's now just `Ok` and `Err` cases and you're only doing anything for `Err`, this can be simplified:

```suggestion
                    Ok(size) => {
                        if let Err(e) = handle_command(&mut device, &serial_buf[..size]) {
                            eprintln!("Unexpected error handling command: {e}");
                        }
```

Reply:
Github was unable to commit this from UI but this change is included in https://github.com/votingworks/vxsuite/pull/4501/commits/2d46774fc3a0e9e0e1a0c6860b0cf2b900713d52